src/client/cli/cmd/launch.cpp

```diff
@@ -268,7 +268,16 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
 
     if (parser->isSet(cpusOption))
     {
-        request.set_num_cores(parser->value(cpusOption).toInt());
+        bool conversion_pass;
+        const int cpu_count = parser->value(cpusOption).toInt(&conversion_pass); // Base 10 conversion with check.
+
+        if (!conversion_pass || cpu_count < 1)
+        {
+            cerr << "error: Please enter a non-zero integer value for the CPU count.\n ";
```

Comment:
Little tweak to match other error msgs and remove the space at the end:

```suggestion
        const auto& cpu_text = parser->value(cpusOption);
        const int cpu_count = cpu_text.toInt(&conversion_pass); // Base 10 conversion with check.

        if (!conversion_pass || cpu_count < 1)
        {
            fmt::print(cerr, "error: Invalid CPU count '{}', need a positive integer value.\n", cpu_text);
```

Reply:
Hey @surahman, notice that `cpu_text` was meant to refer to the text before converting to int, which is what needs to go in the error message (can't use the int from a failed conversion).

Reply:
BTW, you should be able to "Commit suggestion" through GH directly.

Reply:
> BTW, you should be able to "Commit suggestion" through GH directly.

Sorry, I did not know that - will keep it in mind for next time. Diff is out of date now 😞 .

> Hey @surahman, notice that `cpu_text` was meant to refer to the text before converting to in.

🤦🏽 corrected.

Reply:
No worries!